packages/tupaia-web/src/features/EntitySearch/EntityMenu.tsx

```diff
@@ -100,12 +99,21 @@ const EntityMenuItem = ({
         <MenuLink to={link}>
           {entity.name} {entity.type === 'facility' && <HospitalIcon />}
         </MenuLink>
-        <IconButton onClick={onExpand} disabled={parentIsLoading || !nextChildren}>
+        <IconButton
+          onClick={toggleExpanded}
+          disabled={parentIsLoading || !nextChildren}
+          aria-controls="entity-expand-button"
+        >
           <ExpandIcon />
         </IconButton>
       </FlexRow>
       {isExpanded && (
-        <EntityMenu children={nextChildren} projectCode={projectCode} isLoading={isLoading} />
+        <EntityMenu
+          children={nextChildren}
+          projectCode={projectCode}
+          isLoading={isLoading}
+          aria-expanded={isExpanded}
```

Comment:
Just checking - does this get carried down to the menu element?

Reply:
ah very good point. I have moved it to the List element and it's always true there since the menu list only shows when expanded.